### ISSUE TITLE
Broken template openwhisk-nodejs: Action entrypoint 'hello' is not a function.

### ISSUE THREAD

**Issue by sampaxk** [serverless/serverless-openwhisk#202]
I've ran these commands on my system:
`serverless create --template openwhisk-nodejs`
`serverless deploy`

This is the handler.js that has been generated:
```
'use strict';

function hello(params) {
  const name = params.name || 'World';
  console.log('log', { payload: `Hello, ${name}` });
  return { payload: `Hello, ${name}!` };
}

exports.hello = hello;
```

Then I've invoked the function on IBM Cloud Functions dashboard, but the result is an error:
```
Results:
{
  "error": "Initialization has failed due to: Action entrypoint 'hello' is not a function."
}
Logs:
[
  "2021-01-24T12:31:26.570863Z    stderr: Error during initialization: Action entrypoint 'hello' is not a function.",
  "2021-01-24T12:31:26.585Z       stderr: The action did not initialize or run as expected. Log data might be missing."
]
```

I have some functions deployed without the help of serverless framework but only using the ibmcloud CLI, and those functions doesn't experience this issue.
Here is the command that I've used to deploy the functions:
`ibmcloud fn action update notify dist/bundle.js --kind nodejs:12`
Note that the functions that I deploy with ibmcloud CLI are not zipped.

Anyway if I change `exports.hello` to `exports.main` in handler.js and deploy with by this command: `ibmcloud fn action update example handler.js --kind nodejs:12`, the function works seamlessly.

**Here is my system configuration:**
Linux Mint 20

_serverless --version_
Framework Core: 2.20.1 (local)
Plugin: 4.4.2
SDK: 2.3.2
Components: 3.5.1

_node -v_
v15.6.0

_npm -v_
7.4.0

_uname -r_
5.8.0-38-generic

**Here is the example.zip generated by serverless packaging:**
[example.zip](https://github.com/serverless/serverless-openwhisk/files/5862240/example.zip)

**Comment by pgrzesik** [comment]
Hello @sampaxk, thanks for reporting. I'm not expert on `openwhisk` integration to be honest, but I've noticed that you're using node `v15.6.0` that is confirmed to be a cause for invalid `zip` artifacts. Could you verify if on lower node versions (e.g. 15.5.0) the issue still persists?

**Comment by sampaxk** [comment]
Thank you, I've fixed the issue downgrading to the latest NodeJS TLS version: 14.15.4

I've compared the two archives being generated during the packaging process of serverless.
In the first screenshot you see the archive being generated with NodeJS LTS 14.15.4, in the second screenshot there is the one generated with NodeJS 15.6.0.
The difference I see is that there is no information about the original size and the saving in the secondo archive, anyway the data in the archive is not corrupt.

![image](https://user-images.githubusercontent.com/73034222/105642755-f17e8d80-5e8b-11eb-8aeb-0d83c25973d2.png)
![image](https://user-images.githubusercontent.com/73034222/105642767-18d55a80-5e8c-11eb-9e98-83f853e58203.png)
